Humidifier: add purpose-specific triggers (Labs feature) (#45071)
home-assistant/home-assistant.github.io: source/_triggers/humidifier.started_humidifying.markdown

---
title: "Humidifier started humidifying"
trigger: humidifier.started_humidifying
domain: humidifier
description: "Triggers after one or more humidifiers start actively humidifying."
related_triggers:
  - humidifier.turned_on
  - humidifier.started_drying
---

> The **Humidifier started humidifying** trigger fires when a humidifier {% term entity %} begins actively adding moisture to the air. A humidifier that is turned on does not necessarily humidify continuously. It may pause once the target humidity is reached and then resume when the air dries out again. **Humidifier started humidifying** fires every time it moves from idle back into active humidification.

Reviewer: Nitpick: Prefer "can" or "might" instead of "may" to talk about possibility, so readers don't confuse it with permission:

```suggestion
The **Humidifier started humidifying** trigger fires when a humidifier {% term entity %} begins actively adding moisture to the air. A humidifier that is turned on does not necessarily humidify continuously. It can pause when the target humidity is reached and then resume when the air dries out again. **Humidifier started humidifying** fires every time it moves from idle back into active humidification.
```

> The **Humidifier started humidifying** trigger fires when a humidifier {% term entity %} begins actively adding moisture to the air. A humidifier that is turned on does not necessarily humidify continuously. It may pause once the target humidity is reached and then resume when the air dries out again. **Humidifier started humidifying** fires every time it moves from idle back into active humidification.

Author: ```suggestion
The **Humidifier started humidifying** trigger fires when a humidifier {% term entity %} begins actively adding moisture to the air. A humidifier that is turned on does not necessarily humidify continuously. It pauses once the target humidity is reached and then resumes when the air dries out again. **Humidifier started humidifying** fires whenever it moved from idle back to active humidification.
```
